Example/Tests/GINIImageAnalysisNoResultsViewControllerTests.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  GINIImageAnalysisNoResultsViewControllerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 10/16/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+final class GINIImageAnalysisNoResultsViewControllerTests: XCTestCase {
+    
+    let viewControllerTitle = "Title"
+    lazy var viewController: ImageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController(title: self.viewControllerTitle)
+    lazy var mockedItems: [(image: UIImage?, text: String)] = [
+        (self.loadImage(withName: "tabBarIconHelp"), "item 1 text"),
+        (self.loadImage(withName: "tabBarIconHelp"), "item 2 text"),
+        (self.loadImage(withName: "tabBarIconHelp"), "item 3 text"),
+    ]
+    
+    override func setUp() {
+        _ = viewController.view
+    }
+    
+    func testViewControllerTitleOnInitialization() {
+        let vcTitle = viewController.title
+        
+        XCTAssertEqual(viewControllerTitle, vcTitle, "view controller title should be equals to the one passed in the initialization")
```

Comment:
Same as the other pull request - the "equals to the one passed in the initialization" is not really important.

Reply:
I will try to keep it simple for future PRs.